pom.xml

```diff
@@ -214,19 +209,18 @@
 
         <plugins>
             <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-compiler-plugin</artifactId>
-            </plugin>
-            <plugin>
-                <groupId>org.sonatype.plugins</groupId>
-                <artifactId>nexus-staging-maven-plugin</artifactId>
-                <extensions>true</extensions>
+                <groupId>org.sonatype.central</groupId>
```

Comment:
you should now need the `central-publishing-maven-plugin` because it is inherited by the finos-parent-pom, see https://github.com/finos/finos-parent-pom/blob/main/pom.xml#L239

Reply:
The situation here https://github.com/finos/legend-shared/blob/master/legend-shared-test-reports/pom.xml#L47 is that we're trying to skip the deployment of that one module.

For `waitUntil` - let's see how much more time this is adding exactly. We do want the information of when is the artifact published I think, since it's then good to use in downstream projects.